src/functions/Invoke-AzOpsPush.ps1

```diff
@@ -23,6 +23,10 @@
         [string[]]
         $ChangeSet,
 
+        [Parameter(Mandatory = $false, ValueFromPipeline = $true)]
+        [string[]]
+        $DeleteSetContents,
```

Comment:
Why have we added this parameter, the 'DeleteSet' was already in the ChangeSet and should be generic enough?